core/client/app/styles/components/settings-menu.css

```diff
@@ -153,14 +153,20 @@ body.settings-menu-expanded .settings-menu-container {
 /* Background
 /* ---------------------------------------------------------- */
 
-body.settings-menu-expanded .content-cover {
+.settings-menu-expanded .eater-of-clicks-devourer-of-worlds-mother-of-dragons,
+.mobile-menu-expanded .eater-of-clicks-devourer-of-worlds-mother-of-dragons {
     position: absolute;
     top: 0;
     right: 0;
     bottom: 0;
     left: 0;
-    z-index: 600;
+    z-index: 1200;
     transition: transform 0.4s cubic-bezier(0.1, 0.7, 0.1, 1);
-    transform: translate3d(-350px, 0px, 0px);
     /* Not off the screen, to give a parallax effect */
 }
+.settings-menu-expanded .eater-of-clicks-devourer-of-worlds-mother-of-dragons {
+    transform: translate3d(-350px, 0px, 0px);
+}
+.mobile-menu-expanded .eater-of-clicks-devourer-of-worlds-mother-of-dragons {
```

Comment:
Could always shorten it down to '.devouring-mother' - reference to Defiance is less popular but still geeky :grin: